src/storage/txn/scheduler.rs

```diff
@@ -495,6 +499,15 @@ impl<E: Engine, L: LockManager> TxnScheduler<E, L> {
         self.inner.scale_pool_size(pool_size)
     }
 
+    pub fn memory_quota_capacity(&self) -> usize {
+        self.inner.memory_quota.capacity()
+    }
+
+    pub(in crate::storage) fn set_memory_quota_capacity(&self, cap: usize) {
+        SCHED_TXN_MEMORY_QUOTA.allocated.set(cap as i64);
```

Comment:
The label "allocated" actually means the capacity of the total memory can be used by scheduler tasks, so is it better to rename it with "capacity"? The memory is not pre-allocated, the value of "used" should be the amount of allocated memory